### ISSUE TITLE
runMods.R incomplete merge

### ISSUE THREAD

**Issue by karinorman** [karinorman/vital_signs#3]
@katherinesiegel  It looks like the workhorse functions (analysis/src/runMods.R) for running models didn't get properly merged on one of your commits. Can you take a look and see if you remember? Otherwise we can loop Lauren in.

**Comment by katherinesiegel** [comment]
Yup, just fixed the weird merge. Back when we were working on this code in 2017, there was a command (I think it was quartz) that worked on Lauren's computer but not mine (because I don't use a Mac), so we were going back and forth between using quartz() and X11() in Lines 54 and 55, depending on who was using the code. Hopefully there's a more elegant solution now though-- I'll take a look.
